app/lib/rocketchat/services/connect.ts

```diff
@@ -260,7 +260,13 @@ function stopListener(listener: any): boolean {
 
 async function login(credentials: ICredentials, isFromWebView = false): Promise<ILoggedUser | undefined> {
 	// RC 0.64.0
-	await sdk.current.login(credentials);
+	if (sdk.current) {
+		await sdk.current.login(credentials);
+	} else {
+		const { server } = store.getState().server;
+		sdk.initialize(server);
```

Comment:
Awkward to call initialize on login. We should have called it on server select instead of here.